server/pulp/server/async/tasks.py

```diff
@@ -764,20 +764,19 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
         :param kwargs:  Original keyword arguments for the executed task.
         :param einfo:   celery's ExceptionInfo instance, containing serialized traceback.
         """
+        original_formatted_traceback = traceback.format_exc()
```

Comment:
i don;t see traceback variable defined or passed anywhere, should it be einfo.traceback?

Reply:
i see, we have an imported traceback module, nvm then!